File: app/src/main/java/be/hcpl/android/filmtag/model/DataExportFormat.java
package be.hcpl.android.filmtag.model;

import java.util.List;
import java.util.Map;

/**
 * Created by hcpl on 2/08/15.
 */
public class DataExportFormat {

    private List<Roll> rolls;

    private Map<Long, List<Frame>> frames;

    public List<Roll> getRolls() {
        return rolls;
    }

    public void setRolls(List<Roll> rolls) {
        this.rolls = rolls;
    }

    public Map<Long, List<Frame>> getFrames() {
        return frames;
    }

    public void setFrames(Map<Long, List<Frame>> frames) {
        this.frames = frames;
    }
}

File: app/src/main/java/be/hcpl/android/filmtag/model/DataExportFormat.kt
package be.hcpl.android.filmtag.model

import androidx.annotation.Keep

/**
 * Needed only to combine frame and roll data in a single format that can be exported
 */
@Keep
data class DataExportFormat(
    var rolls: List<Roll>? = null,
    var frames: Map<Long, List<Frame>>? = null,
)
